src/main/java/core/basesyntax/ReadWriteList.java

```diff
@@ -10,16 +11,32 @@
     private ReadWriteLock lock = new ReentrantReadWriteLock();
```

Comment:
Consider making the 'lock' variable 'final' as its value is not intended to change 🛠